temporalio/workflow.py

```diff
@@ -986,6 +989,17 @@ def memo_value(
     return _Runtime.current().workflow_memo_value(key, default, type_hint=type_hint)
 
 
+def upsert_memo(updates: Mapping[str, Any]) -> None:
+    """Adds, modifies, and/or removes memos, with upsert semantics.
+
+    Every memo that has a matching key has its value replaced with the one specified in ``updates``.
+    If the value is set to ``None``, the memo is removed instead.
```

Comment:
What if I don't want to remove the key, but rather I want the key to be present but set with a `None` value? `None` is a legitimate payload (of encoding `binary/null`) and is different than the absence of a key.

Reply:
I have since learned that server doesn't support upserting with a key and a null value, null is always assumed as remove-key server side and all non-.NET SDKs represent this as a null value of a map on upsert. So all good here, can ignore me.